### PR TITLE
Update extras to use texture setter, update Text to use getter/setter for style/text

### PR DESCRIPTION
**Pull request by serprex**
I took the liberty to remove setText while making since change since Sprite dropped setTexture

### CODE REVIEW

File: src/text/Text.js [hunk 1]
```diff
@@ -44,8 +44,8 @@ function Text(text, style)
 
     core.Sprite.call(this, core.Texture.fromCanvas(this.canvas));
 
-    this.setText(text);
-    this.setStyle(style);
+    this.text = text;
+    this.style = style;
```
Comment (englercj) [inline src/text/Text.js]: Even though these will technically set the underscored tracker variables, we should put them in the ctor explicitly as well and document them as private members.

File: src/text/Text.js [hunk 2]
```diff
@@ -161,20 +176,15 @@ Text.prototype.updateText = function ()
...
 
-    var outputText = this.text;
-
     // word wrap
     // preserve original text
-    if (this.style.wordWrap)
-    {
-        outputText = this.wordWrap(this.text);
-    }
+    var outputText = this.style.wordWrap ? this.wordWrap(this.text) : this.text;
```
Comment (englercj) [inline src/text/Text.js]: Use `_style` internally to prevent object getter call overhead.

**Comment by englercj** [comment]
Thanks for this! Will merge it as soon as the small notes I had are fixed. Really appreciate the work you have been doing for v3!

**Comment by serprex** [comment]
I don't know if github notifies when a pull request receives new commits

Addressed small notes

**Comment by englercj** [comment]
:+1:

**Pull request merged**